tests/PidTest.cpp

```diff
@@ -0,0 +1,331 @@
+/**
+ * @file PidTest.cpp
+ * @brief PidGainクラス, Pidクラスをテストする
+ * @author miyahara046
+ */
+
+#include "Pid.h"
+#include "gtest/gtest.h"
+namespace etrobocon2025_test {
+
+  // PidGainの正のゲインがそのまま格納されるかをテスト
+  TEST(PidGainTest, PidGainPlus)
+  {
+    double kp = 0.1;
+    double ki = 0.03;
+    double kd = 0.612;
+    double expected_kp = 0.1;
+    double expected_ki = 0.03;
+    double expected_kd = 0.612;
+    PidGain actualPidGain(kp, ki, kd);
+    EXPECT_DOUBLE_EQ(expected_kp, actualPidGain.kp);
+    EXPECT_DOUBLE_EQ(expected_ki, actualPidGain.ki);
+    EXPECT_DOUBLE_EQ(expected_kd, actualPidGain.kd);
+  }
+  // PidGainの0のゲインがそのまま格納されるかをテスト
```

Comment:
空白行が欲しいかな。
他のところではできてるので、ここだけ忘れた感じかな。

```suggestion
  }
  
  // PidGainの0のゲインがそのまま格納されるかをテスト
```